.idea/gradle.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
```

Comment:
Also, can you remove the contents of ./idea/** from this PR. Those changes are not required in the repo.
You can add .idea to a .gitignore file so it gets ignored next time.